vyxal/transpile.py

```diff
@@ -272,31 +269,44 @@ def transpile_structure(struct: structure.Structure, indent: int) -> str:
         return temp
 
     if isinstance(struct, structure.MonadicModifier):
-        element_A = transpile(lambda_wrap(struct.branches[1][0]))
-        return element_A + "\n" + elements.modifiers.get(struct.branches[0])
+        element_A = transpile_ast(
+            [lambda_wrap([struct.branches[1][0]])], indent
+        )
+        return element_A + "\n" + elements.modifiers[struct.branches[0]]
 
     if isinstance(struct, structure.DyadicModifier):
-        element_A = transpile_ast(lambda_wrap(struct.branches[1][0]), indent)
-        element_B = transpile_ast(lambda_wrap(struct.branches[1][1]), indent)
+        element_A = transpile_ast(
+            [lambda_wrap([struct.branches[1][0]])], indent
+        )
+        element_B = transpile_ast(
+            [lambda_wrap([struct.branches[1][1]])], indent
+        )
         return (
             element_A
             + "\n"
             + element_B
             + "\n"
-            + indent_str(elements.modifiers.get(struct.branches[0]), indent)
+            + indent_str(elements.modifiers[struct.branches[0]], indent)
         )
     if isinstance(struct, structure.TriadicModifier):
-        element_A = transpile(lambda_wrap(struct.branches[1][0]))
-        element_B = transpile(lambda_wrap(struct.branches[1][1]))
-        element_C = transpile(lambda_wrap(struct.branches[1][2]))
+        element_A = transpile_ast(
+            [lambda_wrap([struct.branches[1][0]])], indent
+        )
+        element_B = transpile_ast(
+            [lambda_wrap([struct.branches[1][1]])], indent
+        )
+        element_C = transpile_ast(
+            [lambda_wrap([struct.branches[1][2]])], indent
+        )
         return (
             element_A
             + "\n"
             + element_B
             + "\n"
             + element_C
             + "\n"
-            + elements.modifiers.get(struct.branches[0])
+            + elements.modifiers[struct.branches[0]]
         )
 
     raise ValueError(struct)
+    raise ValueError(struct)
```

Comment:
be more specific with this error? (also why is it here twice?)

Reply:
That seems like a code duplication error